From b657bd3e5fb28a4a428e9acbb473cd3cba5b9f91 Mon Sep 17 00:00:00 2001
From: =?utf-8?q?Diego=20Elio=20'Flameeyes'=20Petten=C3=B2?= <flameeyes@gmail.com>
Date: Thu, 21 May 2009 12:17:02 +0200
Subject: [PATCH 1/2] Don't redeclare isblank().

While technically isblank() is a C library function, nothing stops it from
being a macro, and indeed it seems to be on glibc-2.10.

This should not be a problem because ctype.h already declares it on
probably all the systems where it's used.
---
 envy24control/strstr_icase_blank.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/envy24control/strstr_icase_blank.c b/envy24control/strstr_icase_blank.c
index af44854..4c325cf 100644
--- a/envy24control/strstr_icase_blank.c
+++ b/envy24control/strstr_icase_blank.c
@@ -48,7 +48,6 @@ int strstr_icase_blank(const char * const string1, const char * const string2)
 	char search_string[MAX_SEARCH_FIELD_LENGTH];
 	char *pstr;
 	int pos_first_non_blank;
-	int isblank(int c);
 
 	strncpy(search_string, string2, MAX_SEARCH_FIELD_LENGTH);
 	search_string[MAX_SEARCH_FIELD_LENGTH - 1] = '\0';
-- 
1.6.3.1

